reana_workflow_controller/k8s.py

```diff
@@ -398,3 +399,81 @@ def delete_k8s_ingress_object(ingress_name, namespace):
             "Exception when calling ExtensionsV1beta1->"
             "Api->delete_namespaced_ingress: {}\n".format(k8s_api_exception)
         )
+
+
+def create_dask_dashboard_ingress(cluster_name, workflow_id):
+    """Create K8S Ingress object for Dask dashboard."""
+    # Define the middleware spec
+    middleware_spec = {
+        "apiVersion": "traefik.containo.us/v1alpha1",
+        "kind": "Middleware",
+        "metadata": {"name": f"replacepath-{workflow_id}", "namespace": "default"},
+        "spec": {
+            "replacePathRegex": {
+                "regex": f"/{workflow_id}/dashboard/*",
+                "replacement": "/$1",
+            }
+        },
+    }
+
+    ingress = client.V1Ingress(
+        api_version="networking.k8s.io/v1",
+        kind="Ingress",
+        metadata=client.V1ObjectMeta(
+            name=f"dask-dashboard-ingress-{cluster_name}",
```

Comment:
BTW is dashboard exposure working already? I'm getting 404 page not found when trying out locally.